vault/templates.cli.md

```diff
@@ -2,12 +2,41 @@
 id: 39menlcx30lpo1ci4px7aew
 title: CLI
 desc: ''
-updated: 1654385553923
+updated: 1655414733603
 created: 1654385553923
 templateId: "39menlcx30lpo1ci4px7aew"
 ---
 ## Summary
 
-## Options
+{{fm.desc}}
 
-## Examples
\ No newline at end of file
+## Usage
+
+```sh
+example --help output
+
+Options:
+  --version         Show version number                                [boolean]
+    ...
+```
+
+### Options
+<!-- Key value list for any non-obvious options, with sub lists as need for allowed values and definitions of them e.g.
+    - `--output`: controls how note is formatted
+        - values: `json|md_gfm|md_dendron`
+            - `json`: JSON output
+            - `md_dendron`: dendron markdown
+            - `md_gfm`: github flavored markdown 
+-->
+
+### Actions
```

Comment:
nitpick: actions is not a universal property of CLIs. we've also not been good about calling this the same thing consistently (actions/methods/commands). how about calling this commands? i actually don't think we do this anywhere but its more in line with traditional unix cli nomenclature

Reply:
You're right, there seems to be two methods that are used across the current CLI commands (word after `dendron`), there's positional commands (e.g. `dev` and `init` in `dendron publish`) and actions (as an option, e.g. `createMissingLinkedNotes` in `dendron doctor`), I conflated the two in my head.

I agree with changing it to commands and referring to the positionals (where they exist), actions should have been listed under the options header instead. 

Just thinking style-wise the best approach then for each option - I think it could go a 4th level header for each option (instead of the list in there now), then details in there so something like

```markdown
### Options
#### `--output`
Controls how note is formatted

values:
- `json`: JSON output
- `md_dendron`: dendron markdown
- `md_gfm`: github flavored markdown 
```
or for the doctor command
```markdown
### Options
#### `--action`
What action the doctor should take

values: 
- fixFrontmatter: Currently unsupported, do not use.
- h1ToTitle: ![[dendron.topic.doctor#h1totitle,1:#*]]
- h1ToH2: ![[dendron.topic.doctor#h1ToH2,1:#*]]
- removeStubs: ![[dendron.topic.doctor#removeStubs,1:#*]]
- createMissingLinkedNotes: ![[dendron.topic.doctor#createMissingLinkedNotes,1:#*]]
- regenerateNoteId: ![[dendron.topic.doctor#regenerateNoteId,1:#*]]
- findBrokenLinks: ![[dendron.topic.doctor#findBrokenLinks,1:#*]]
- fixRemoteVaults: ![[dendron.topic.doctor#fixRemoteVaults,1:#*]]
- fixAirtableMetadata: ![[dendron.topic.doctor#fixAirtableMetadata,1:#*]]
- addMissingDefaultConfigs: ![[dendron.topic.doctor#addMissingDefaultConfigs,1:#*]]
- removeDeprecatedConfigs: ![[dendron.topic.doctor#removeDeprecatedConfigs,1:#*]]
- fixSelfContainedVaultsInConfig: Fixes issues in the `dendron.yml` when using/moving to a self-contained vault
```

Reply:
sounds good to me :)